plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/AppEngineStandardRuntimeChangeListener.java

```diff
@@ -107,7 +106,6 @@ public void handleEvent(IFacetedProjectEvent event) {
     }
 
     // Add the App Engine facet
-    IProject project = facetedProject.getProject();
```

Comment:
Does getProject actually do nothing? I have this funny feeling that it may cause something to happen via side effects even of we don't sue the return value.